### PR TITLE
add instant tests

### PR DESCRIPTION
**Pull request by sowu880**
Add instant tests for datetime shift utility.

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Utility/DateTimeUtilityTests.cs [hunk 1]
```diff
@@ -199,6 +225,50 @@ public void GivenADateTimeWithoutDayOrAgeOver89_WhenDateShift_ThenDateTimeShould
...
+        }
+
+        [Theory]
+        [MemberData(nameof(GetInstantDataForDateShiftButShouldBeRedacted))]
+        public void GivenAnInstantWithAgeOver89_WhenDateShift_ThenInstantShouldBeRedacted(Instant instant, string expectedInstantString)
+        {
+            var node = ElementNode.FromElement(instant.ToTypedElement());
+            var processResult = DateTimeUtility.ShiftDateTimeAndInstantNode(node, string.Empty, string.Empty, true);
+
+            Assert.Equal(expectedInstantString?.ToString() ?? null, node.Value);
```
Comment (moria97) [inline src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Utility/DateTimeUtilityTests.cs:268]: we can remove "?? null" here? same for other places.

Comment (moria97) [reply src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Utility/DateTimeUtilityTests.cs:268]: Also remove the toString method

Comment (moria97) [inline src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Utility/DateTimeUtilityTests.cs:268]: should be Assert.Equal(expectedInstantString, node.Value?.ToString())

File: src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Utility/DateTimeUtilityTests.cs [hunk 2]
```diff
@@ -199,6 +225,50 @@ public void GivenADateTimeWithoutDayOrAgeOver89_WhenDateShift_ThenDateTimeShould
...
         }
 
+        [Theory]
+        [MemberData(nameof(GetInstantDataForRedact))]
+        public void GivenAnInstant_WhenRedact_ThenInstantShouldBeRedacted(Instant instant, string expectedInstantString)
+        {
+            var node = ElementNode.FromElement(instant.ToTypedElement());
+            var processResult = DateTimeUtility.RedactDateTimeAndInstantNode(node, true);
+
+            Assert.Equal(expectedInstantString?.ToString() ?? null, node.Value);
```
Comment (skywalker-wu) [inline src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Utility/DateTimeUtilityTests.cs:235]: equals to  "expectedInstantString?.ToString()"?

**Review by moria97 — APPROVED** [review]

**Review by skywalker-wu — APPROVED** [review]

**Review by BoyaWu10 — APPROVED** [review]

**Pull request merged**